### PR TITLE
Scala 2.13.10 release notes draft

### PR DESCRIPTION
**Pull request by SethTisue**
[link to rendered version](https://github.com/SethTisue/scala-dev/blob/scala-2.13.10-release-notes/2.13.10.md)

Feedback welcome

### CODE REVIEW

**Review by Philippus — APPROVED** [review]

File: 2.13.10.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Upgrading from 2.12? Enable `-Xmigration` while upgrading to request migration advice from the compiler.
+
+## Contributors
+
+A big thank you to everyone who's helped improve Scala by reporting bugs, improving our documentation, spreading kindness in discussions around Scala, and submitting and reviewing pull requests! You are all magnificent.
+
+We especially acknowledge and thank A. P. Marki, also known as Som Snytt, who is responsible for an especially large share of the improvements in this release.
+
+This release was brought to you by 7 contributors, according to `git shortlog -sn --no-merges @ ^v2.13.8 ^2.12.x`. Thank you A. P. Marki, Liang Yan, Seth Tisue, Antoine Parent, Luc Henninger, liang3zy22, 梦境迷离.
```
Comment (liang3zy22) [inline 2.13.10.md]: "Liang Yan" is my name. "liang3zy22" is my github user name. I think using one of two is enough. Then reduce to 6 contributors.

Comment (SethTisue) [reply 2.13.10.md]: will do 👍

Comment (SethTisue) [reply 2.13.10.md]: (I’ll fix it in .mailmap so it stays fixed)

Comment (SethTisue) [reply 2.13.10.md]: https://github.com/scala/scala/pull/10187

**Comment by SethTisue** [comment]
The notes are now live at https://github.com/scala/scala/releases/tag/v2.13.10

**Pull request closed without merging**